hardhat.config.js

```diff
@@ -35,11 +35,18 @@ const optimizerSettings = {
     },
 };
 
+const outputSelectionSettings = {
+    "*": {
+        "*": ["storageLayout"],
```

Comment:
can you exclude test contracts, and the build artifacts column?

Reply:
I looked up `outputSelection` option in detail, it enables outputs of storageLayout but does not have option to exclude test contracts and build artifacts.

https://docs.soliditylang.org/en/v0.8.20/using-the-compiler.html#input-description